test/clustertest/tests/TimingTest.cpp

```diff
@@ -52,7 +52,7 @@ struct TimingTest : tpunit::TestFixture {
             uint64_t processTime = SToUInt64(result["processTime"]);
             uint64_t totalTime = SToUInt64(result["totalTime"]);
 
-            // Only leader is expected to have these set.
+            // Leader should have peek and process times, followers only peek.
```

Comment:
It's weird to me that we thought this was fine before, that `peek` times should be zero on followers for any escalated commands.